soft-delete/Program.cs

```diff
@@ -145,28 +138,28 @@ private static async Task SoftDeleteTest()
         // Helper method to retrieve retrieve the CloudBlobClient object in order to interact with the storage account.
         // The retry policy on the CloudBlobClient object is set to an Exponential retry policy with a back off of 2 seconds
         // and a max attempts of 10 times.
-        private static CloudBlobClient GetCloudBlobClient()
+        private static BlobServiceClient GetCloudBlobClient()
         {
             try
             {
-                CloudStorageAccount storageAccount = CloudStorageAccount.Parse(CloudConfigurationManager.GetSetting("StorageConnectionString"));
-                CloudBlobClient blobClient = storageAccount.CreateCloudBlobClient();
-                IRetryPolicy exponentialRetryPolicy = new ExponentialRetry(TimeSpan.FromSeconds(2), 10);
-                blobClient.DefaultRequestOptions.RetryPolicy = exponentialRetryPolicy;
+                BlobClientOptions blobClientOptions = new BlobClientOptions();
+                blobClientOptions.Retry.Delay = TimeSpan.FromSeconds(1);
+                blobClientOptions.Retry.MaxRetries = 1;
```

Comment:
Should we just remove these? Also now the comment above should be updated to match.

Reply:
Done.